src/containers/AboutMeContainer/AboutMe.js

```diff
@@ -1,29 +1,31 @@
-import React, {useState} from 'react';
+import React, {useState, useEffect} from 'react';
+import axios from 'axios'
 import AboutMeContent from "../../components/AboutMeComponent/AboutMeContent.js";
 import Title from "../../components/Title/Title";
 import classes from "../AboutMeContainer/AboutMe.module.css";
-import InfoAboutMe from '../AboutMeContainer/InfoAboutMe.json'
-
 
 const AboutMe = () => {
-  const [aboutMeInfo] = useState(InfoAboutMe)
+  const [aboutMeData, setAboutMeData] = useState([])
+  const fetchData = async () => {
+    const response = await axios.get('https://api.codecarver.dev/article/3')
```

Comment:
Let's put the ID of the article in a constant outside of the component, later this ID should come from an environment variable, for now it's enough with having it in a constant.